Speedykom/Regional-Pandemic-Analytics
# Feat/vault service handled by nginx

## Description ##
This PR enables access to vault service via custom domain for each environment (dev -local -prod )
 ## Testing Instructions ##
-Start the vault container by make start-vault
-generate env files by ./gen-env.sh ( chmod +x ./gen-env.sh if needed before )
- After running the project in its corresponding environment , access : 
- https://vault.igad.local for local env
- https://vault2.igad-health.eu for dev env
- https://vault.igad-health.eu for prod env
- Vault console should be accessible to login via token
